src/launch.js

```diff
@@ -1,15 +1,19 @@
+import 'react-app-polyfill/ie11';
+import 'react-app-polyfill/stable';
 import FHIR from 'fhirclient';
 
 // retrieve launch context from backend, if configured
-let context_url = `${process.env.PUBLIC_URL}/launch-context.json`;
+let context_url = process.env.PUBLIC_URL + '/launch-context.json';
```

Comment:
Just curious- does IE not support string formatting with back-ticks?

Reply:
Sorry, Ivan, I believe so, I was debugging and forgot to change back -  but it should work as is without the backticks.